src/platform_impl/macos/monitor.rs

```diff
@@ -230,6 +230,10 @@ impl MonitorHandle {
                 return None;
             }
 
+            if time.time_value < 1 {
+                return None;
+            }
+
```

Comment:
Can we use [`checked_div()`](https://doc.rust-lang.org/std/primitive.i64.html#method.checked_div) instead?